8.x/core-concepts/cruds.md

```diff
@@ -2,99 +2,55 @@
 
-In this tutorial, the principles of CRUD operation in Larammerce will be taught, so, let's follow a few steps to create it:
+#### Make migrations
 
-**Step 1**: Create Migration
+Larammerce is a schema-first platform. So you must create a table for your todos.
 
-**Step 2**: Add Model
+Run the following command in the terminal:
 
-**Step 3**: Add Controller
-
-**Step 4**: Add Veiws 
-
-::: warning WARNING
-The tutorials in this document are based on Laravel **version 8.75**.
-:::
-
-For example, a to-do list is created to show how to the operation.
-
-### Step 1: Create Migration
-Since the Laravel framework is a **schema-first** framework, so, the database schema must be created first.
-At this migration, so open a terminal or command line and run the following command:
-
-```bash
+```bash{1}
 php artisan make:migration create_todos_table
 ```
 
-After this command, a file will be placed in the `database/migrations` path, also, the following code should be placed in the migration file to create a `todos` table.
+Insert these codes in the path `/larammerce/database/migrations/[some_numbers]_create_todos_table.php`:
 
-##### EXAMPLE
-```php{17-24}
+```php{7,8}
 <?php
-
-use Illuminate\Database\Migrations\Migration;
-use Illuminate\Database\Schema\Blueprint;
-use Illuminate\Support\Facades\Schema;
-
-class CreateTodosTable extends Migration
-{
-    /**
-     * Run the migrations.
-     *
-     * @return void
-     */
+    ...
     public function up()
     {
         Schema::create('todos', function (Blueprint $table) {
-            #ID column
-            $table->id(); 
-            #subject column
+            $table->id();
             $table->string("subject");
-            #status column
             $table->tinyInteger("status");
-            #create_at and update_at column
             $table->timestamps();
         });
     }
-
-    /**
-     * Reverse the migrations.
-     *
-     * @return void
-     */
-    public function down()
-    {
-        Schema::dropIfExists('todos');
-    }
-}
+    ...
+   
 ```
 
-Now, enter this command in the terminal to run the migration:
+Run the following command in the terminal:
 
-```bash
+```bash{1}
 php artisan migrate
 ```
 
-:::tip NOTE
-You can check your database to make sure the table `todos` is created.
-:::
+You can check your database to make sure the table `todos` is created:
+
+![todos_table](/img-cruds/40.png)
+
+#### Create Todo model
 
-### Step 2: Add  Model
-At this stage, a Laravel model should be created, so for this purpose, create a file in the path `app/Models/Todo.php`
-and put some properties in the `Todo` class as follows:
+Create the file `Todo.php` in the path `/larammerce/app/Models/` and put these codes inside:
 
-##### EXAMPLE
-```php{7-13,16,18-20,22-24,26-28,30,31,33-36}
+```php{1-36}
 <?php
```

Comment:
if you explain the usage of these functions and their relation to the baseModel and the fact that we can overwrite functions and customize them,  it would be a better proposal.